presto-parquet/src/main/java/io/prestosql/parquet/ParquetTimestampUtils.java

```diff
@@ -16,12 +16,17 @@
 import com.google.common.annotations.VisibleForTesting;
 import com.google.common.primitives.Ints;
 import com.google.common.primitives.Longs;
+import io.prestosql.parquet.predicate.ParquetRangeStatistics;
+import io.prestosql.parquet.predicate.TupleDomainParquetPredicate;
+import io.prestosql.plugin.base.type.DecodedTimestamp;
+import io.prestosql.plugin.base.type.PrestoTimestampEncoder;
 import io.prestosql.spi.PrestoException;
+import io.prestosql.spi.predicate.Domain;
 import org.apache.parquet.io.api.Binary;
 
-import java.util.concurrent.TimeUnit;
-
 import static io.prestosql.spi.StandardErrorCode.NOT_SUPPORTED;
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.joda.time.DateTimeConstants.SECONDS_PER_DAY;
```

Comment:
Add a constant to io.prestosql.spi.type.Timestamps and use here.
This will make it easier to do https://github.com/prestosql/presto/issues/5168 later